app/_data/docs_nav_kuma_dev.yml

```diff
@@ -19,9 +19,29 @@ items:
             url: "/introduction/how-kuma-works/#vm-and-k8s-support"
           - text: Kuma vs XYZ
             url: "/introduction/how-kuma-works/#kuma-vs-xyz"
-      - text: Enterprise
-        url: /introduction/enterprise/
-      - text: Changelog
+      - text: Architecture
+        url: /introduction/architecture/
+        items:
+          - text: Components
+            url: "/introduction/architecture/#components"
+          - text: Kubernetes mode
+            url: "/introduction/architecture/#kubernetes-mode"
+          - text: Universal mode
+            url: "/introduction/architecture/#universal-mode"
```

Comment:
```suggestion
```
I think you can remove these if you want since they're just links to sections on this page.

Reply:
Good idea. Im going to close this comment so I can grab the `item` tag too.